configure.ac

```
@@ -1,6 +1,6 @@
 dnl The line below must be formatted AC_INIT(opam,VERSION) with no extra spaces
-AC_INIT(opam,2.0.0)
-AC_COPYRIGHT(Copyright 2012-2017 OcamlPro SAS)
+AC_INIT(opam,2.1.0~beta)
```

```suggestion
AC_INIT(opam,2.1.0~alpha)
```
(seems better to have development versions reporting that, until an actual beta is released?). `alpha` is of course better than `dev`, since a `beta` release is then naturally a greater version.